x-pack/plugins/ml/server/routes/job_service.ts

```diff
@@ -548,7 +551,9 @@ export function jobServiceRoutes({ router, routeGuard }: RouteInitialization) {
           aggFieldNamePairs,
           splitFieldName,
           splitFieldValue,
-          runtimeMappings
+          runtimeMappings,
+
```

Comment:
Should we get rid of this extra line?

Reply:
https://github.com/elastic/kibana/pull/91830/commits/8ca21bb9de9c182d0c95e40cd550f4596af8f8e0